common/app/views/fragments/page/email/body.scala.html

```diff
@@ -13,7 +13,6 @@
                                         <tr>
                                             <td class="no-pad">
                                                 @content
-                                                @fragments.email.support(page)
```

Comment:
yep good point, it's the current hardcoded support ask - it will be inserted via Braze now. I can delete the template too